### PR TITLE
Home-category

### PR DESCRIPTION
**Pull request by Chakravarthi0**
Added the following:
-home page
-Category page
-rules page
-questionpage and
-result page

preview: https://sharp-visvesvaraya-5ad2d0.netlify.app/

image
image
image
image
image

### CODE REVIEW

**Review by sakshamAK — APPROVED** [review]
Nicely done

File: pages/question/question.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+.option:hover{
+    background-color: var(--gray-light);
+}
+
+.next-btn{
+    transform: translateY(-50%);
+    margin-top: 5rem;
+    font-size: 2rem;
+    padding: 1rem;
+    margin-bottom: 3rem;
```
Comment (msdeshmukh009) [inline pages/question/question.css]: You can also do `margin:5rem 0 3rem;` if I'm not wrong.

**Review by Varu98 — APPROVED** [review]

File: pages/rules/rules.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+.rules-list {
+  width: 50%;
+  max-width: max-content;
+  background-color: var(--white-pure);
+  margin-top: 5rem;
+}
+
+.rules-head {
+  text-align: center;
+  font-size: 3.5rem;
```
Comment (msdeshmukh009) [inline pages/rules/rules.css]: I see you are using `font-size:3.5rem;` in lots of places so consider making a utility class or something. That will be very useful in many scenarios. And same for `text-align:center;`.

**Review by msdeshmukh009 — APPROVED** [review]
Looking good!!

File: pages/rules/rules.css (hunk 2 above)
Comment (Chakravarthi0) [reply pages/rules/rules.css]: Sure, I will do it.

**Comment by preethasri** [comment]
UI looks good 👍

**Pull request merged**